src/components/image/image.html

```diff
@@ -67,10 +62,10 @@ <h3>Round Image</h3>
         </div>
 
         <div class="component-container">
-            <div class="artsy-img img-round ">
+            <div class="artsy-img img-round w-30">
                 <img src="https://picsum.photos/800/400" alt ="random images" loading="lazy" />
```

Comment:
better text for alt tag such as component image is more descriptive for screen readers